tests/utils.go

```diff
@@ -2048,7 +2048,7 @@ func KubevirtFailHandler(message string, callerSkip ...int) {
 
 	for _, ns := range []string{metav1.NamespaceSystem, NamespaceTestDefault} {
 		// Get KubeVirt specific pods information
-		pods, err := virtClient.CoreV1().Pods(ns).List(metav1.ListOptions{LabelSelector: "kubevirt.io"})
+		pods, err := virtClient.CoreV1().Pods(ns).List(metav1.ListOptions{LabelSelector: "kubevirt.io,cdi.kubevirt.io"})
```

Comment:
That means that the pods need both labels. :/

Reply:
oops, i rushed this. Will fix.